packages/noco-docs/content/en/index.md

```diff
@@ -89,7 +85,7 @@ We use ``Lerna`` to manage multi-packages. We have the following [packages](http
 
 ## Contributions
 
-All contributions are welcome. NocoDB projects can be found in the [NocoDB](https://github.com/nocodb) Github organization. Our core uses [The AGPL V3 license](https://github.com/nocodb/nocodb/blob/master/LICENSE) and all contributors should read the [contribution guides](https://github.com/nocodb/nocodb/tree/master/scripts/contribute) before making your first contribution. 
+All contributions are welcome. NocoDB projects can be found in the [NocoDB](https://github.com/nocodb) Github organization. Our core uses [The AGPL V3 license](https://github.com/nocodb/nocodb/blob/master/LICENSE) and all contributors should read the [contribution guides](https://github.com/nocodb/nocodb/blob/master/.github/CONTRIBUTING.md) before making your first contribution. 
```

Comment:
Cool, please confirm if we are not referring to ./contribute folder from docs as such too.